### ISSUE TITLE
A screenshot for every theme

### ISSUE THREAD

**Issue by adegas-pfx** [arbennett/jupyterlab-themes#112]
A screen shot of every theme would greatly help the visibility of the project :)

**Comment by zjwilliams20** [comment]
Agreed. I'm too lazy to install each one individually unless I know it's something I'd want.
